compiler/src/dmd/func.d

```diff
@@ -119,9 +119,9 @@ private struct FUNCFLAG
     bool safetyInprocess;    /// working on determining safety
     bool nothrowInprocess;   /// working on determining nothrow
     bool nogcInprocess;      /// working on determining @nogc
-    bool returnInprocess;    /// working on inferring 'return' for parameters
+    bool scopeInprocess;     /// infer `return` and `scope` for parameters
     bool inlineScanned;      /// function has been scanned for inline possibilities
-    bool inferScope;         /// infer 'scope' for parameters
+    bool dummy;              /// unused
```

Comment:
If you wonder 'Why not just remove this?', you're in for a treat. Follow up PR coming soon.

Reply:
Can be removed after https://github.com/dlang/dmd/pull/16752 is merged

Reply:
Or maybe not if older compilers are affected?

Reply:
Between versions, there's no expectation of binary compatibility of AST nodes right?